### PR TITLE
Concise latest items

### PR DESCRIPTION
**Pull request by sumeetadur**
This provides the option for displaying concise items in the stream. Concise items are displayed without a tag and subheading.

### CODE REVIEW

**Comment by callumlocke** [comment]
This looks good to me.

@andygnewman
@i-love-robots

File: components/card/index.js [hunk 1]
```diff
@@ -40,7 +45,7 @@ module.exports = React.createClass({
 						<h3 className="n-card__headline">
 							<a className="n-card__headline__link" href={`/content/${item.id}${item.referrerTracking ? item.referrerTracking : ''}`} data-trackable="headline">{item.title}</a>
 						</h3>
-						<div className="n-card__subheading">{item.subheading}</div>
```
Comment (i-like-robots) [inline components/card/index.js]: Does this not simply collapse when there is no content?

Comment (sumeetadur) [inline components/card/index.js]: Yes it does, but it leaves a 4px margin-top so thought it was better to remove the element completely so the headline could be exactly vertically-aligned in the middle

**Comment by sumeetadur** [comment]
Closing in favour of https://github.com/Financial-Times/next-stream-page/pull/887

**Pull request closed without merging**